infra/gcp/ensure-static-ips.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+#
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script creates static ip addresses for services deployed in
+# the `aaa` cluster
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
+. "${SCRIPT_DIR}/lib.sh"
+
+PROJECT_NAME="kubernetes-public"
+
+function usage() {
+    echo "usage: $0" > /dev/stderr
+    echo > /dev/stderr
+}
+
+if [ $# != 0 ]; then
+    usage
+    exit 1
+fi
+
+function create_address() {
+    if [ $# != 1 ]; then
+        echo "create_address(name) requires 1 argument" >&2
+        return 1
+    fi
+    local address_name="$1"
+
+    # Create address if it doesn't exist
+    if ! gcloud compute addresses describe "$address_name" --global \
+        --project "$PROJECT_NAME" > /dev/null 2>&1; then
+        gcloud compute addresses create "$address_name" --global \
+            --project "$PROJECT_NAME"
+        echo "Address: $address_name created successfully"
+    else
+        echo "Address: $address_name already exists"
+    fi
+}
+
+ADDRESSES=(
+    "gcsweb-k8s-io"
+    "k8s-io-ingress-canary"
+    "k8s-io-ingress-prod"
+    "perf-dash-k8s-io-ingress-prod"
+    "slack-infra-ingress-prod"
+)
+
+for address in "${ADDRESSES[@]}"; do
+    color 6 "Create address: $address"
+    create_address "$address"
```

Comment:
```suggestion
    ensure_address "$address"
```

Reply:
Moved the function to the `lib.sh` file, and addressed the name change